packages/jest-haste-map/src/HasteFS.ts

```diff
@@ -33,7 +33,7 @@ export default class HasteFS {
 
   getDependencies(file: Config.Path): Array<string> | null {
     const fileMetadata = this._getFileData(file);
-    return (fileMetadata && fileMetadata[H.DEPENDENCIES]) || null;
+    return (fileMetadata && fileMetadata[H.DEPENDENCIES].split('\t')) || null;
```

Comment:
Could consider extracting the delimiter to a constant somewhere (regardless of its value). Becomes self-documenting by having a name